### ISSUE TITLE
Stratos cannot use MariaDB forge servers

### ISSUE THREAD

**Issue by krutten** [blacksmith-community/mariadb-forge-boshrelease#8]
Before the credentials were added, Stratos would show

```
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] Attempting to parse VCAP_SERVICES
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] Found 0 database service instances
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] No DB Environment detected - skipping migrations
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] Configuration loaded.
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] Stratos Version: 2.5.0
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] HTTP client initialized.
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] Encryption key set.
   2019-10-22T13:19:35.76-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:35 UTC 2019] Found 0 database service instances
   2019-10-22T13:19:36.69-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:36 UTC 2019] ========================
   2019-10-22T13:19:36.69-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:36 UTC 2019] = Stratos DB Migration =
   2019-10-22T13:19:36.69-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:36 UTC 2019] ========================
   2019-10-22T13:19:36.69-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:36 UTC 2019] Database provider: sqlite3
   2019-10-22T13:19:36.69-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:19:36 UTC 2019] Current 0
```

After the credentials change, it's not finding the version
```
   2019-10-22T13:01:03.86-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:01:03 UTC 2019] Attempting to parse VCAP_SERVICES
   2019-10-22T13:01:03.86-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:01:03 UTC 2019] Found 1 database service instances
   2019-10-22T13:01:03.86-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:01:03 UTC 2019] Using first database service instance: stratosdb
   2019-10-22T13:01:03.86-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:01:03 UTC 2019] Attempting to apply Cloud Foundry database service config from credentials
   2019-10-22T13:01:03.86-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:01:03 UTC 2019] Applied Cloud Foundry database service config (provider: mysql)
   2019-10-22T13:01:03.86-0700 [APP/PROC/WEB/0] OUT INFO[Tue Oct 22 20:01:03 UTC 2019] Migrating mysql instance on 10.250.106.114
   2019-10-22T13:01:05.89-0700 [APP/PROC/WEB/0] OUT FATA[Tue Oct 22 20:01:05 UTC 2019] Migration failed! Failed to get database version: dial tcp 10.250.106.114:3306: connect: connection refused
```

**Comment by krutten** [comment]
Update, when creating the DB, it errors on
```
   2019-10-23T10:18:12.56-0700 [APP/PROC/WEB/0] OUT INFO[Wed Oct 23 17:18:12 UTC 2019] Running migration: 20181129140500_UserFavorites
   2019-10-23T10:18:12.57-0700 [APP/PROC/WEB/0] OUT INFO[Wed Oct 23 17:18:12 UTC 2019] Running migration: 20190305144600_EndpointSubtype
   2019-10-23T10:18:15.10-0700 [APP/PROC/WEB/0] OUT INFO[Wed Oct 23 17:18:15 UTC 2019] Running migration: 20190515133200_AuthEndpoint
   2019-10-23T10:18:15.55-0700 [APP/PROC/WEB/0] OUT INFO[Wed Oct 23 17:18:15 UTC 2019] Running migration: 20190522121200_LocalUsers
   2019-10-23T10:18:15.57-0700 [APP/PROC/WEB/0] OUT ERRO[Wed Oct 23 17:18:15 UTC 2019] Apply() failed:Error 1465: Triggers can not be created on system tables
   2019-10-23T10:18:15.57-0700 [APP/PROC/WEB/0] OUT FATA[Wed Oct 23 17:18:15 UTC 2019] Migration failed! Error 1465: Triggers can not be created on system tables
```

**Comment by jhunt** [comment]
'Tis fixed as of #10 (now merged)
